packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -20,25 +20,51 @@ const double _kDrawerHeaderHeight = 160.0 + 1.0; // bottom edge
 /// See also:
 ///
 ///  * [Drawer]
+///  * [UserAccuntsDrawerHeader], a variant of [DrawerHeader] that is
+///    specialized for showing user accounts.
 ///  * [DrawerItem]
 ///  * <https://www.google.com/design/spec/patterns/navigation-drawer.html>
-
 class DrawerHeader extends StatelessWidget {
   /// Creates a material design drawer header.
   ///
   /// Requires one of its ancestors to be a [Material] widget.
   const DrawerHeader({
     Key key,
     this.decoration,
+    this.padding: const EdgeInsets.only(
+                    top: 16.0,
+                    left: 16.0,
+                    right: 16.0,
+                    bottom: 8.0
+                  ),
+    this.duration: const Duration(milliseconds: 250),
+    this.curve: Curves.ease,
```

Comment:
We should audit, but I'm pretty sure all these ease covers should be fastInSlowOut everywhere in the material library.